shell/browser/api/electron_api_web_contents.cc

```diff
@@ -535,7 +535,7 @@ void WebContents::InitWithSessionAndOptions(
   managed_web_contents()->GetView()->SetDelegate(this);
 
   auto* prefs = web_contents()->GetMutableRendererPrefs();
-  prefs->accept_languages = g_browser_process->GetApplicationLocale();
+  SetAcceptLanguages(prefs);
```

Comment:
Can you move the language preferences code into separate files and then reference it here? Like this:

```c++
#include "shell/common/lang_util.h"

prefs->accept_languages = GetSystemLanguage();
```

It would make the code easier to maintain, and we might want to reuse the code in other places.

Reply:
I wasn't sure where to put this kind of code. Extracted: d947ed9e7db679576d92045b7bdfe8254b37f645